src/metabase/server/middleware/session.clj

```diff
@@ -45,6 +46,19 @@
 (def ^:private ^String metabase-session-timeout-cookie  "metabase.TIMEOUT")
 (def ^:private ^String anti-csrf-token-header           "x-metabase-anti-csrf-token")
 
+(defmulti session-cookie-name
```

Comment:
Is this open to extension? This being a `defmulti` feels weird.

Reply:
Eh i guess it makes sense because of the other stuff. You should add a default case that throws though i suppose.

Reply:
You're right, I can't see a future where we add another session type with a different cookie name. I'll change it to a function with a `case` expression.